Doc/pages/analysis.rst

```diff
@@ -409,8 +409,6 @@ design of therapeutic compounds.
 Virtual Instruments
 ^^^^^^^^^^^^^^^^^^^
 
-^^^^^^^^^^^^^^^^^^^
```

Comment:
I added this in the last PR so that there was a line under each section see Infrared for example.

![image](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/334f39fa-e16f-4420-b9dd-d02b57a15910)

It is quite faint can you see it on your screen?

Reply:
OK, I will bring that one back. I can just about make out the line on my screen.